tutorials/migration.md

```diff
@@ -463,78 +391,71 @@ cp <openHAB 1.x conf>/configurations/sitemaps/* <openHAB 2 conf>/sitemaps/*
 
   * Static webview files are now located in `<openhab 2 conf>/html` instead of  `/usr/share/openhab/webapps`
 
-  * The name of the sitemap (i.e. the word right after `sitemap` at the top of the file) must match the file name. For example the file named `myhome.sitemap` should start with `sitemap myhome`.
+  * The name of the sitemap (i.e. the word right after `sitemap` at the top of the file) must match the file name. 
+  For example the file named `myhome.sitemap` should start with `sitemap myhome`.
   * Charts no longer display a legend by default.
   If this parameter is not set, the legend is hidden if there is only one chart series.
   Add 'legend=true' to restore the chart's legend. 
 
 ### Rules
-  * All references to org.openhab.core.* in imports and class references should be removed. All of these classes are automatically included and have moved.
-  * import org.joda.time.* statements should also be removed.
-  * To test for Items with an Undefined state replace `if(MyItem.state == Undefined)` with `if(MyItem.state == NULL)`. Case matters. `NULL` is only valid when testing an Item's state to see if it is undefined. `null` is used pretty much everywhere else to mean "no value" and usually indicates no result or an error.
-  * The `HSBType` state type can no longer be constructed using a `java.awt.Color` object, and there is no longer a `toColor()` method.  Use the following alternatives:
+  * All references to org.openhab.* in imports and class references should be removed. All of these classes are automatically included and have moved.
+  * import org.joda.time.* statements should also be removed, they are also included by default.
+  * To test for Items with an Undefined state replace `if(MyItem.state == Undefined)` with `if(MyItem.state == NULL)`. 
+  Case matters. 
+  `NULL` is only valid when testing an Item's state to see if it is undefined. 
+  `null` is used pretty much everywhere else to mean "no value" and usually indicates no result or an error.
+  * The `HSBType` state type can no longer be constructed using a `java.awt.Color` object, and there is no longer a `toColor()` method. 
+  Use the following alternatives:
 
 ```java
 var HSBType hsb = HSBType::fromRGB(color.red, color.green, color.blue)
 var Color color = Color::getHSBColor(hsb.hue.floatValue / 360, hsb.saturation.floatValue / 100, hsb.brightness.floatValue / 100)
 ```
 
-Continue watching the logs as you move files over. There will likely be a number
-of errors. Take note of them with plans to return and correct them if they
-persist.
+Continue watching the logs as you move files over. 
+There will likely be a number of errors. 
+Take note of them with plans to return and correct them if they persist.
 
 ## Testing
 
 - **Items**: Watch `<openHAB 2 userdata>/logs/events.log` and methodically work through all
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.